.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  kinetic:
+    docker:
+      - image: autonomoustuff/docker-builds:kinetic-perception
+    environment:
+      MAKEFLAGS: -j1
+      COLCON_WORKERS: 2
+    steps:
+      - checkout
+      - run:
+          name: Set Up Container
+          command: |
+            git submodule update --init --recursive
+            apt update -qq && apt install -y libmosquitto-dev gksu python3-pip python3-setuptools
+            pip3 install -U setuptools
+            apt install -y python3-colcon-common-extensions
+      - run:
+          name: Install ROS Dependencies
+          command: |
+            source `find /opt/ros -name setup.bash | sort | head -1`
+            cd ros/src
+            wstool init
+            test -f "dependencies.rosinstall" && wstool merge "dependencies.rosinstall"
+            wstool up
```

Comment:
I'm not familiar with wstool, but why is it needed?
I would have assumed a regular rosdep update and install as directed in the [install docs](https://github.com/autowarefoundation/autoware/wiki/Source-Build) would be fine.